ibridgesgui/browser.py

```diff
@@ -488,3 +485,41 @@ def _metadata_edits(self, operation):
                 elif operation == "delete":
                     meta.delete(new_key, new_val, new_units)
                 self._fill_metadata_tab(irods_path)
+
+    def _fs_select(self, path_select):
+        """Retrieve the path (file or folder) from a QFileDialog
+           path_select: PyQt6.QtWidgets.QFileDialog.getExistingDirectory
+                        PyQt6.QtWidgets.QFileDialog.getOpenFileName
+
+        """
+        yes_button = PyQt6.QtWidgets.QMessageBox.StandardButton.Yes
+        no_button = PyQt6.QtWidgets.QMessageBox.StandardButton.No
+        if isinstance(path_select, tuple):
+            path = path_select[0]
+            reply = PyQt6.QtWidgets.QMessageBox.question(self,
+                                     'Upload File', path,
+                                     yes_button | no_button, no_button)
+        else:
+            path = path_select
+            reply = PyQt6.QtWidgets.QMessageBox.question(self,
+                                     'Upload Folder', path,
```

Comment:
This dialog is a little bit on the sparse side. (Also the upload file) Perhaps at least tell them their file will be uploaded to this destination?

Reply:
I changed the info to:
```
Upload File:
/Users/staig001/Demo

to
/nluu12p/home/research-test-christine

Only new data will be added.
```
If `overwrite` is checked the last sentence will switch to: `All data will be updated.`